common/src/main/java/com/redhat/cloud/notifications/models/Endpoint.java

```diff
@@ -33,6 +33,8 @@
 @JsonNaming(SnakeCaseStrategy.class)
 public class Endpoint extends CreationUpdateTimestamped {
 
+    public static final String[] SORT_FIELDS = {"id", "name", "enabled", "endpoint_type", "created"};
```

Comment:
Will Hibernate be happy with `endpoint_type`?

Reply:
No - I just updated the code to make hibernate happy